static/src/javascripts/projects/common/modules/commercial/contributions-service.js

```diff
@@ -337,7 +372,7 @@ export const renderBanner = (response) => {
                     container,
                     Banner,
                     { submitComponentEvent, ...module.props},
-                    true
+                    isPuzzlesBanner ? false : true
```

Comment:
Could this just be `!isPuzzlesBanner`?

I also think maybe we should have an explanatory comment here about the `CacheProvider` issue.

Reply:
👍